### PR TITLE
Enable slashing protection reference tests

### PR DESCRIPTION
**Pull request by rain-on**

### CODE REVIEW

File: slashing-protection/referencetests/src/test/java/tech/pegasys/web3signer/slashingprotection/ReferenceTestRunner.java [hunk 1]
```diff
@@ -88,36 +90,66 @@ public void cleanup() {
...
+              }
+            });
+
+        // web3signer doesn't allow for partial imports, so - if it is expected, then
+        // expect import to throw.
+        if (step.isShouldSucceed()) {
+          slashingProtection.importData(
+              new ByteArrayInputStream(interchangeContent.getBytes(UTF_8)));
+          verifyImport(step, gvr);
+        } else {
```
[the remark this answers is not in the document]
Comment (jframe) [reply slashing-protection/referencetests/src/test/java/tech/pegasys/web3signer/slashingprotection/ReferenceTestRunner.java:135]: Had a look at some test cases and single_validator_slashable_attestations_double_vote has should_succeed = true and allow_partial_import = true. Should additional checks be done for a case like this?

File: slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java [hunk 2]
```diff
@@ -201,49 +201,51 @@ private void importAttestations(
...
         LOG.warn(
-            "Attestation {} of validator {} surrounds an existing entry",
-            i,
-            validator.getPublicKey());
-      }
-
-      if (attestationValidator.alreadyExists()) {
-        LOG.debug(
-            "Attestation {} for validator {} already exists in database, not imported",
+            "Attestation #{} for validator {} - source is great than target epoch",
```
Comment (jframe) [inline slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java:207]: /great/greater

Comment (jframe) [inline slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java:207]: nit: is the "Attestation #{}" format deliberate? elsewhere is just "Attestation {}"

File: slashing-protection/referencetests/src/test/java/tech/pegasys/web3signer/slashingprotection/ReferenceTestRunner.java [hunk 3]
```diff
@@ -88,36 +90,66 @@ public void cleanup() {
...
-    final TestFileModel model = objectMapper.readValue(inputFile.toFile(), TestFileModel.class);
-    final String interchangeContent =
-        objectMapper.writeValueAsString(model.getInterchangeContent());
-
-    slashingProtection.importData(new ByteArrayInputStream(interchangeContent.getBytes(UTF_8)));
-
-    verifyImport(model);
+    setup();
+    try {
+      System.out.println(inputFile.toString());
```
[the remark this answers is not in the document]
Comment (rain-on) [reply slashing-protection/referencetests/src/test/java/tech/pegasys/web3signer/slashingprotection/ReferenceTestRunner.java:113]: removed

File: slashing-protection/referencetests/src/test/java/tech/pegasys/web3signer/slashingprotection/ReferenceTestRunner.java [hunk 4]
```diff
@@ -59,11 +60,12 @@
           .registerModule(new InterchangeModule())
           .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
   private final ValidatorsDao validators = new ValidatorsDao();
+  private final MetadataDao metadataDao = new MetadataDao();
   private EmbeddedPostgres slashingDatabase;
   private String databaseUrl;
```
[the remark this answers is not in the document]
Comment (rain-on) [reply slashing-protection/referencetests/src/test/java/tech/pegasys/web3signer/slashingprotection/ReferenceTestRunner.java:65]: done

File: slashing-protection/referencetests/src/test/java/tech/pegasys/web3signer/slashingprotection/ReferenceTestRunner.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (rain-on) [reply slashing-protection/referencetests/src/test/java/tech/pegasys/web3signer/slashingprotection/ReferenceTestRunner.java:135]: I have no idea what partial_import implies - so I'm leaving it out for now.

File: slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java (hunk 2 above)
Comment (rain-on) [reply slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java:207]: done

Comment (rain-on) [reply slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java:207]: cleaned up to be more meaningful.

**Review by jframe — APPROVED** [review]

**Pull request merged**